### DISCUSSION TITLE
Installation problem PK-Sim on Windows 11 on Mac (with Parallels)

### DISCUSSION THREAD

**Discussion by MillaKerpenkath** [Open-Systems-Pharmacology/Forum#1542] [Questions & Problems] [answered]
Hello, 
I have just tried downloading OSP Suite (Version 11 Update 2) to my Mac with Parallels (Windows 11). The installation works without any problems. But as soon as I open the program I get the following error message: "Unable to load DLL 'OSPSuite.FuncParser Native.dI|': The specified module was not found. (Exception from HRESULT: 0×8007007E)".
Does anyone know a solution for this problem?

![IMG_0732](https://github.com/Open-Systems-Pharmacology/Forum/assets/134547725/ba922e21-70e7-43cb-8d8d-17a7b9e83f3d)

**Comment by Yuri05** [comment]
On your Parallels Windows installation, start the command prompt (**CMD**) and enter **wmic OS get OSArchitecture**
What is the output?
![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/e45e1b99-80aa-4234-b4e3-738ad2eb819e)

**Reply by MillaKerpenkath** [reply]
The output is:

C:\Windows\System32>wmic OS get OSArchitecture
OSArchitecture
64-Bit-ARM-Prozessor


C:\Windows\System32>

<img width="495" alt="Bildschirmfoto 2023-05-24 um 23 30 14" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/134547725/ecf13018-f8ca-4bf4-8613-dcf5686103c0">

**Reply by Yuri05** [reply]
**ARM64** architecture is not supported by the OSP Suite. 
Only **x64** architecture is supported. You need to install the x64 version of Windows.

**Reply by FunkyBeatMaster** [reply]
Hi, I have the same problem. Did you know you cannot install a non ARM version of windows on a computer with ARM motherboards ? 
Is it possible to use the source code to compile it into an ARM functioning version ? ARM based PC have arrived on the market and will be spreading soon.

**Comment by FunkyBeatMaster** [comment, marked as answer]
As an answer to myself: I installed the multipack C++ installer (https://mpvci.co.uk) and now it works. Cheers to all recent mac users: use parallel 11, Windows 11, and this download. 
![works](https://github.com/user-attachments/assets/8a06d3e1-6124-45f7-9847-18ea63247b55)

**Reply by msevestre** [reply]
WOW. @Yuri05 Maye this should be part of our documentation?

**Reply by Yuri05** [reply]
That's really cool @FunkyBeatMaster, thanks for the update! 👍

**Comment by rwmcintosh** [comment]
The only thing missing was the C++ redistributable linked by @FunkyBeatMaster. 
There is an x86-64 emulator in ARM Windows.

We can give it a try and verify it works.

**Comment by rwmcintosh** [comment]
Confirmed - Parallels running ARM Windows

When I used the installer, I had the same issue as OP.

I installed the latest [VC Redist](https://aka.ms/vs/17/release/vc_redist.x64.exe)

PK-Sim starts and simulations can run.

**Comment by FunkyBeatMaster** [comment]
I knew there is a x64 emulator in Win11ARM, but it still didn't work…
I found other programs having same issue, and the solution was VC++ redistributable.
Glad my stubbornness helped you guys !
Happy SIM.
